images/runtime/php-fpm/8.3/base.bullseye.Dockerfile

```diff
@@ -0,0 +1,137 @@
+FROM oryxdevmcr.azurecr.io/private/oryx/php-fpm-8.3-bullseye
```

Comment:
where is `base.<debian-version>.Dockerfile` used vs `8.3.<debian-version>.Dockerfile`

Reply:
Resolving this, discussed offline.